chapters/equations.tex

```diff
@@ -817,13 +817,19 @@ \section{Initialization, initial equation, and initial algorithm}\label{initiali
 variables) is equal.  Often, this is a nonlinear system of equations and therefore it may be necessary to provide appropriate guess values (i.e., \lstinline!start! values and
 \lstinline!fixed = false!) in order to compute a solution numerically.
 
-It may be difficult for a user to figure out how many initial equations have to be added, especially if the system has a higher index. A tool may add or remove initial equations
-automatically such that the resulting system is structurally nonsingular.  In these cases diagnostics are appropriate since the result is not unique and not necessarily what the user
-expects.  A missing initial value of a discrete-time variable which does not influence the simulation result, may be automatically set to the \lstinline!start! value or its default without
-informing the user.  For example, variables assigned in a \lstinline!when!-clause which are not accessed outside of the \lstinline!when!-clause and where \lstinline!pre! is not explicitly
-used on these variables, do not have an effect on the simulation.
+It may be difficult for a user to figure out how many initial equations have to be added, especially if the system has a higher index.
 \end{nonnormative}
 
+Based on these non-normative considerations we have the following rules:
+A tool may add or remove initial equations automatically according to the rules below such that the resulting system is structurally nonsingular.
```

Comment:
Before an item list like this, we almost always use a colon instead of period:
```suggestion
A tool may add or remove initial equations automatically according to the rules below such that the resulting system is structurally nonsingular:
```